ast/compile.go

```diff
@@ -1277,6 +1284,39 @@ func (c *Compiler) getExports() *util.HashMap {
 	return rules
 }
 
+func (c *Compiler) checkDuplicateImports() {
+	if !c.strict {
+		return
+	}
+
+	for _, name := range c.sorted {
+		mod := c.Modules[name]
+		processedImports := map[Var]*Import{}
```

Comment:
Map iteration in golang is randomised, do we need to take care in some way so that errors are stable?  Hmmm I don't think so, looking at what's next, since we're only checking for existence in line 1311. Or am I missing something?

Reply:
Correct, we only care to check for pre-existing imports here. The order of `mod.Imports` determines the order of the errors.